docs/source/dataframe_schemas.rst

```diff
@@ -230,6 +231,42 @@ Since ``required=True`` by default, missing columns would raise an error:
     1  pandera
 
 
+.. _column validation:
+
+Stand-alone Column Validation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In addition to being used in the context of a ``DataFrameSchema``, ``Column``
+objects can also be used to validate columns in a dataframe on its own:
+
+.. testcode:: dataframe_schemas
+
+    import pandas as pd
+    import pandera as pa
+
+    from pandera import Column, Check
+
+    df = pd.DataFrame({
+        "column1": [1, 2, 3],
+        "column2": ["a", "b", "c"],
+    })
+
+    column1 = Column(pa.Int, name="column1")
+    column2 = Column(pa.String, name="column2")
+
+    # pass the dataframe as an argument to the Column object callable
+    df = column1(df)
+    validated_df = column2(df)
+
+    # use the DataFrame.pipe the method to validate two columns
+    validated_df = df.pipe(column1).pipe(column2)
+
+
+For multi-column use cases, the ``DataFrameSchema`` is still recommended, but
+if you have one or a small number of columns to verify, using ``Column``
+objects by themselves is appropriate.
+
```

Comment:
Another usage that I expected to see documented is:

Columns can also be validated on their own by using: `column2_schema.validate(df["column2"])`